### PR TITLE
Add a new portfolio 'Color Changing'

### PR DESCRIPTION
**Pull request by happyhearthacker**
Add a new portfolio 'Color Changing'. Update that in index.html
Hoping you will accept my PR and give me 'Hacktoberfest-accepted' label

### CODE REVIEW

File: Color changing/blue.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+;root{
+	--mainColor;#15202B:
```
Comment (jofftiquez) [inline Color changing/blue.css:2]: Use 2 spaces for tabs. Apply to all

File: Color changing/bucket-policy.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (jofftiquez) [inline Color changing/bucket-policy.txt:1]: Is this file required?

File: Color changing/green.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
+:root{
+	--mainColor:#606B56;
```
Comment (jofftiquez) [inline Color changing/green.css:2]: Use 2 spaces for tabs. Apply to all

File: Color changing/index.html [hunk 4]
```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
```
Comment (jofftiquez) [inline Color changing/index.html:1]: Use 2 spaces for tabs. Apply to all

File: Color changing/index.html [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+	</section>
+
+	<section class="s2">
+		<div class="main-container">
+
+			<div class="about-wrapper">
+				<div class="about-me">
+					<h4>More about me</h4>
+
+					<p>I build new projects just to tickle my brain and love teaching others how they're made.</p>
```
Comment (jofftiquez) [inline Color changing/index.html:82]: Use lorem ipsum, no need to add specific details

File: Color changing/index.html [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+
+					<p>While I keep busy teaching courses, I still take interviews in search of a great team & projects that interest me.</p>
+
+
+					<hr>
+
+					<h4>TOP EXPERTISE</h4>
+
+					<p>Fullstack developer with primary focus on Django + React: <a target="_blank" href="resume.pdf">Download Resume</a></p>
```
Comment (jofftiquez) [inline Color changing/index.html:91]: Just a blank link is okay, this is just a template afterall

File: Color changing/index.html [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+
+					<div id="skills">
+						<ul>
+							<li>Python</li>
```
Comment (jofftiquez) [inline Color changing/index.html:95]: Use lorem, no need for specif details

File: Color changing/index.html [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+					</div>
+
+				</div>
+
+				
+				<div class="social-links">
+					<img id="social_img" src="images/follow.jpg">
+					<h3>Find me on Twitter & Youtube</h3>
+
+					<a target="_blank" href="https://www.youtube.com/c/dennisivy">YouTube: @DennisIvy</a>
```
Comment (jofftiquez) [inline Color changing/index.html:119]: Remove specific details, replace with dummy or blank links

File: Color changing/index.html [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+					<br>
+					<a target="_blank" href="https://twitter.com/dennisivy11">Twitter: @DennisIvy11</a>
```
Comment (jofftiquez) [inline Color changing/index.html:121]: Remove specific details, replace with dummy or blank links

File: Color changing/index.html [hunk 10]
```diff
@@ -0,0 +1,198 @@
...
+
+	<section class="s1">
+		<div class="main-container">
+			<h3 style="text-align: center;">Some of my past projects</h3>
+
+			<div class="post-wrapper">
+
+				<div>
+					<div class="post">
+						<img class="thumbnail" src="images/dash.jpg">
```
Comment (jofftiquez) [inline Color changing/index.html:136]: Dummy projects are okay, remove specific projects, you can also use fake photos or placeholders

File: Color changing/post.html [hunk 11]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
```
Comment (jofftiquez) [inline Color changing/post.html:1]: What is this page for?

File: Color changing/script.js [hunk 12]
```diff
@@ -0,0 +1,40 @@
+console.log('Its working')
```
Comment (jofftiquez) [inline Color changing/script.js:1]: Remove debug logs

File: Color changing/script.js [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+
+let theme = localStorage.getItem('theme')
```
Comment (jofftiquez) [inline Color changing/script.js:3]: Always use `const` over `let` if the variable is not mutable.

File: Color changing/script.js [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+
+if(theme == null){
+	setTheme('light')
+}else{
+	setTheme(theme)
+}
+
+let themeDots = document.getElementsByClassName('theme-dot')
```
Comment (jofftiquez) [inline Color changing/script.js:11]: Always use `const` over `let` if the variable is not mutable.

File: Color changing/script.js [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+
+
+for (var i=0; themeDots.length > i; i++){
+	themeDots[i].addEventListener('click', function(){
+		let mode = this.dataset.mode
```
Comment (jofftiquez) [inline Color changing/script.js:16]: Always use `const` over `let` if the variable is not mutable.

File: Color changing/script.js (hunk 12 above)
Comment (jofftiquez) [reply Color changing/script.js:1]: Use 2 spaces for tabs

**Review by jofftiquez — CHANGES_REQUESTED** [review]
Thank you for this. These are my comments:

In general, we avoid themes with specific brandings/person name. Instead we should use placeholders or lorem text. You can promote your name in footer or somewhere by adding "Theme made by [your name]" or similar

**Comment by happyhearthacker** [comment]
Ok, I am doing this.

File: Color changing/post.html (hunk 11 above)
Comment (happyhearthacker) [reply Color changing/post.html:1]: This is to showcase the projects from portfolio.

**Comment by happyhearthacker** [comment]
Now please accept the PR. And give 'Hacktoberfest-accepted' label

**Review by jofftiquez — CHANGES_REQUESTED** [review]
Not all comments are resolved yet, thanks.

**Comment by jofftiquez** [comment]
> Now please accept the PR. And give 'Hacktoberfest-accepted' label

I'm sorry but you have to resolve all review comments before I can accept the PR.

**Comment by happyhearthacker** [comment]
> Hello @happyhearthacker any updates on this? Thanks.

@jofftiquez Just give me 3-4 days. I will resolve all the issues.
Thank you

**Comment by jofftiquez** [comment]
Closing this for inactivity
